src/jenkins.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=C0302
```

Comment:
What is this rule please? I think you can call it by name as well. Also, we usually ask for an explanatory comment as to why disabling is reasonable in this case please, do you think that is applicable here?

Reply:
I renamed this to its name. I think the name is giving enough context that we don't have to elaborate further.